conftest.py

```diff
@@ -73,6 +155,19 @@ def patch_xml():
     return good_xml
 
 
+@pytest.fixture()
+def test_settings():
```

Comment:
If I do this, what will happen if these settings are in conflict with Django's `@override_settings` test decorator? Just want to make sure autouse doesn't end up creating more problems...

Reply:
That's a good question. I think you'd probably have to test to see the order in which things get applied since pytest is doing some weird things under the hood. You can just leave it how it is, if you want.

Reply:
Ok, I'll explore that at a later point in time. Thanks!